src/System.ServiceModel.Primitives/pkg/System.ServiceModel.Primitives.pkgproj

```diff
@@ -18,6 +18,7 @@
     <InboxOnTargetFramework Include="MonoTouch10" />
     <InboxOnTargetFramework Include="net45" />
     <InboxOnTargetFramework Include="win8" />
+    <InboxOnTargetFramework Include="wp8" />
```

Comment:
The error when I built it :).  We pull the inbox information from the targeting packs that were previously shipped, so you could go look at that if you wanted, but the build error should tell you what's going on.